core/templates/dev/head/base.html

```diff
@@ -33,6 +33,9 @@
 <!DOCTYPE html>
 <html ng-app="oppia" ng-controller="Base" itemscope itemtype="http://schema.org/Organization">
   <head>
+    {% block prerender %}
```

Comment:
Also note that if you add a prerender block in this file it will be around for all pages which don't have "block prerender" set, since they extend base.html.